### PR TITLE
Add OpenVINO EP shared lib to Py Wheel

### PR DESCRIPTION
**Pull request by smkarlap**
**Description**: Include the libonnxruntime_providers_openvino.so/.dll to the whl package.

**Motivation and Context**
- OpenVINO EP is now built as a separate shared library, and needs to be explicitly included in the .whl package for python builds

### CODE REVIEW

**Comment by jywu-msft** [comment]
@RyanUnderhill

File: setup.py [hunk 1]
```diff
@@ -164,8 +164,8 @@ def run(self):
   libs.extend(['libonnxruntime_providers_tensorrt.so'])
   # OpenVINO Libs
   if package_name == 'onnxruntime-openvino':
-    if platform.system() == 'Linux':
-      libs.extend(['libovep_ngraph.so'])
+    libs.extend(['libonnxruntime_providers_openvino.so'])
```
Comment (jywu-msft) [inline setup.py:167]: should we just follow the pattern above with DNNL and TensorRT EP instead? (and add comment)

File: setup.py [hunk 2]
```diff
@@ -186,6 +186,8 @@ def run(self):
   libs.extend(['onnxruntime_providers_tensorrt.dll'])
   # DirectML Libs
   libs.extend(['directml.dll'])
+  # OpenVINO Libs
+  libs.extend(['onnxruntime_providers_openvino.dll'])
```
Comment (jywu-msft) [inline setup.py:190]: same comment as above. and should we add for Mac OSX?

Comment (smkarlap) [reply setup.py]: We don't support Mac OSX yet. So not adding one for now.

**Review by snnn — APPROVED** [review]

File: setup.py (hunk 2 above)
Comment (jywu-msft) [reply setup.py]: ok. but openvino does support mac osx. we can add an item to backlog.

**Review by jywu-msft — APPROVED** [review]

File: setup.py (hunk 2 above)
Comment (smkarlap) [reply setup.py]: yes, we will.

**Pull request merged**